server/initializers/constants.ts

```diff
@@ -16,7 +16,7 @@ let config: IConfig = require('config')
 
 // ---------------------------------------------------------------------------
 
-const LAST_MIGRATION_VERSION = 340
+const LAST_MIGRATION_VERSION = 341
```

Comment:
Use 345 so we can insert other migrations in patch versions

Reply:
(changed for 375)